sdk/src/Core/Amazon.Runtime/Internal/_bcl45+netstandard/SSOServiceClientHelpers.cs

```diff
@@ -21,14 +27,25 @@ public static class SSOServiceClientHelpers
 #endif
         )
         {
-            return CreateClient<ICoreAmazonSSOOIDC>(
-                region,
-                ServiceClientHelpers.SSO_OIDC_SERVICE_CLASS_NAME,
-                ServiceClientHelpers.SSO_OIDC_SERVICE_CONFIG_NAME,
-                ServiceClientHelpers.SSO_OIDC_ASSEMBLY_NAME,
-                proxySettings);
+            ICoreAmazonSSOOIDC coreSSO = GlobalRuntimeDependencyRegistry.Instance.GetInstance<ICoreAmazonSSOOIDC>(ServiceClientHelpers.SSO_OIDC_ASSEMBLY_NAME, ServiceClientHelpers.SSO_OIDC_SERVICE_CLASS_NAME, new CreateInstanceContext(new SSOOIDCClientContext(region)));
```

Comment:
Does proxySettings need to be passed to the context here?

Reply:
Done